imports/client/ui/components/timeLine/TimeLine.jsx

```diff
@@ -35,7 +42,68 @@ export default class TimeLine extends React.Component {
 
   openMaxDatePicker = () => {
     this.refs.maxDatePicker.focus()
+    //console.log('maxDAte')
+  }
+PauseOrClear = (event ) => {
+
+
+   window.clearInterval(this.tid);
+  if (this.playOrResumeTrig == 1){
+    //if paused before :clean
+    if (this.PauseOrClearTrig ==1) {
+      k = Math.round(this.props.ui.minTime)
+      this.props.updateUI({currentSliderTime :  k })
+      this.PauseOrClearTrig= 0
+      this.playOrResumeTrig =0
+      }
+    else {
+
+      this.PauseOrClearTrig= 1
+      }
+    }
+  else {
+    this.PauseOrClearTrig= 1
+    this.playOrResumeTrig =0
   }
+ }
+
+  PlayOrResume = (event ) => {
```

Comment:
by convention, caps on first letters are for Class only. functions are lowercase, even in camelCase